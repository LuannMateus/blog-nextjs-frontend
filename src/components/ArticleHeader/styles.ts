import styled, { css } from 'styled-components';
import { Title as HeadingStyle } from '../Heading/styles';

export const Wrapper = styled.section`
  ${({ theme }) => css`
    padding-bottom: ${theme.spacings.xlarge};
    margin-bottom: ${theme.spacings.xlarge};
    border-bottom: 0.1rem solid ${theme.colors.mediumGray};

    ${HeadingStyle} {
      margin: 0;
      margin-bottom: ${theme.spacings.medium};
    }
  `}
`;

export const Excerpt = styled.p`
  ${({ theme }) => css`
    font-size: ${theme.font.sizes.medium};
    margin: ${theme.spacings.medium} 0;
  `}
`;

export const Cover = styled.img`
  ${({ theme }) => css`
    display: block;
    max-width: 100%;
    margin-bottom: ${theme.spacings.medium};
  `}
`;
