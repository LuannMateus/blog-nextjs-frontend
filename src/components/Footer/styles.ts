import styled, { css } from 'styled-components';

export const Container = styled.footer`
  ${({ theme }) => css`
    width: 100%;

    text-align: center;

    a {
      color: inherit;
      text-decoration: none;
      font-size: ${theme.font.sizes.small};
    }
  `}
`;
