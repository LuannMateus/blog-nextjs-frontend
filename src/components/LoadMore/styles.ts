import styled, { css } from 'styled-components';

export const Wrapper = styled.button`
  ${({ theme }) => css`
    width: 16rem;

    border: 1px solid ${theme.colors.darkText};

    background: ${theme.colors.primary};
    color: ${theme.colors.white};

    display: flex;
    justify-content: center;
    align-items: center;

    padding: ${theme.spacings.small};

    cursor: pointer;

    transition: filter 300ms ease-in-out;

    &:hover {
      filter: brightness(50%);
    }

    &:disabled {
      opacity: 0.4;
    }
  `}
`;
