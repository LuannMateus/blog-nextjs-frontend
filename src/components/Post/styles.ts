import styled, { css } from 'styled-components';

export const Wrapper = styled.section`
  ${({ theme }) => css`
    width: 100%;

    padding-left: ${theme.spacings.small};
  `}
`;
