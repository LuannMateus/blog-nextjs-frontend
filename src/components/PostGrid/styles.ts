import styled, { css } from 'styled-components';

export const Wrapper = styled.main`
  ${({ theme }) => css`
    max-width: 100%;
    width: 100%;

    margin: 0 auto;

    display: grid;
    grid-template-columns: repeat(auto-fill, minmax(28rem, 1fr));
    gap: ${theme.spacings.large};

    padding: ${theme.spacings.large};

    @media ${theme.media.lteSmall} {
      grid-template-columns: 1fr;
    }
  `}
`;

export const NotFound = styled.p`
  ${({ theme }) => css`
    text-align: center;
    font-size: ${theme.font.sizes.small};
    padding: ${theme.spacings.large};

    @media ${theme.media.lteSmall} {
      padding: 0;
    }
  `}
`;
