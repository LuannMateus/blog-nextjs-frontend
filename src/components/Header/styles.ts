import styled, { css } from 'styled-components';
import { Title as HeadingStyles } from '../Heading/styles';

export const Wrapper = styled.header`
  ${({ theme }) => css`
    max-width: ${theme.sizes.max};
    width: 100%;

    display: flex;
    justify-content: center;
    align-items: center;

    padding-left: ${theme.spacings.small};
    padding-bottom: ${theme.spacings.xhuge};
    color: ${theme.colors.darkText};
    font-size: ${theme.font.sizes.small};
    margin: 0 auto;

    ${HeadingStyles} {
      margin-right: calc(${theme.spacings.small} - 1rem);
    }

    @media ${theme.media.lteMedium} {
      margin: 0;

      flex-direction: column;
    }
  `}
`;

export const Content = styled.section`
  ${({ theme }) => css`
    display: flex;
    flex-flow: column wrap;
    margin-left: ${theme.spacings.medium};

    @media ${theme.media.lteMedium} {
      text-align: center;

      margin-top: ${theme.spacings.small};
    }
  `}
`;
