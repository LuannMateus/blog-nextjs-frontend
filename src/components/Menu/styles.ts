import styled, { css } from 'styled-components';
import { Container as LogoLinkStyles } from '../LogoLink/styles';

export const Wrapper = styled.nav`
  ${({ theme }) => css`
    width: 27rem;
    height: 100vh;

    display: flex;
    flex-direction: column;
    align-items: center;
    background-color: #000;
    border-radius: 4px;

    padding-top: 1.4rem;

    position: fixed;
    left: -26rem;

    transition: all 300ms ease-in-out;

    z-index: 1000;

    .menuIcon {
      position: relative;
      left: 1.5rem;

      background: #000;
      padding: 0;
    }

    svg {
      width: 3rem;
      color: ${theme.colors.secondary};

      align-self: flex-end;

      padding-right: 1.6rem;

      cursor: pointer;
      transition: filter 300ms ease-in-out;
    }

    svg:hover {
      filter: brightness(50%);
    }

    ${LogoLinkStyles} {
      margin-top: ${theme.spacings.xhuge};
    }

    &.active {
      left: 0rem;
    }
  `}
`;

export const MenuLinkContainer = styled.section`
  ${() => css`
    width: 90%;

    margin-top: 4rem;
  `}
`;
