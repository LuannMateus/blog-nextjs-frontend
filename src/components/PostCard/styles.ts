import styled, { css } from 'styled-components';
import { Title as HeadingStyles } from '../Heading/styles';

export const Wrapper = styled.section`
  ${({ theme }) => css`
    ${HeadingStyles} {
      margin: 0;
      margin-top: calc(${theme.spacings.small} - 0.6rem);
      margin-bottom: ${theme.spacings.small};
    }

    a {
      text-decoration: none;
      color: inherit;
      transition: all 300ms ease-in-out;
    }

    img {
      transition: filter 300ms ease-in-out;
      width: 100%;
      max-width: 100%;
    }

    &:hover a {
      color: ${theme.colors.secondary};
    }

    &:hover img {
      filter: brightness(80%);
    }
  `};
`;

export const Cover = styled.img`
  max-width: 100%;
  transition: opacity 300ms ease-in-out;
`;

export const Excerpt = styled.p``;
