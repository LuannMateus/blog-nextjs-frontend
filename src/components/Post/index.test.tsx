import { screen } from '@testing-library/react';
import { renderTheme } from '../../styles/renderTheme';
import { Post } from '.';

import props from './mock';
import { formatDate } from '../../utils/formatData';

describe('<Post />', () => {
  it('Should render header, excerpt, cover, metadata and post', () => {
    const { container } = renderTheme(<Post {...props} />);

    expect(
      screen.getByRole('heading', { name: props.title }),
    ).toBeInTheDocument();
    expect(screen.getByRole('img', { name: props.title })).toBeInTheDocument();
    expect(
      screen.getByText(/^Fugit aliquip erroribus eu eam/i),
    ).toBeInTheDocument();
    expect(screen.getByText(formatDate(props.createdAt))).toBeInTheDocument();
    expect(
      screen.getByText(/^Blockquote: Et sed legere rationibus/i),
    ).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
