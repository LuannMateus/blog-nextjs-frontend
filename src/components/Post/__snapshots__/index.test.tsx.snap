// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[
  `<Post /> Should render header, excerpt, cover, metadata and post 1`
] = `
.c4 {
  line-height: 1.2;
  color: #000000;
  font-size: 6.4rem;
  text-transform: none;
}

.c2 {
  padding-bottom: 4.0rem;
  margin-bottom: 4.0rem;
  border-bottom: 0.1rem solid #DDDDDD;
}

.c2 .c3 {
  margin: 0;
  margin-bottom: 2.4rem;
}

.c5 {
  font-size: 2.4rem;
  margin: 2.4rem 0;
}

.c6 {
  display: block;
  max-width: 100%;
  margin-bottom: 2.4rem;
}

.c7 {
  font-size: 1.6rem;
  color: #AAAAAA;
  font-style: italic;
}

.c7 .categories span::after {
  content: ', ';
}

.c7 .categories span:last-child::after {
  content: '';
}

.c7 a {
  color: #dc143c;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c7 a:hover {
  -webkit-filter: brightness(50%);
  filter: brightness(50%);
}

.c9 pre[class*='language-'],
.c9 code[class*='language-'] {
  color: #d4d4d4;
  font-size: 13px;
  text-shadow: none;
  font-family: Menlo,Monaco,Consolas,'Andale Mono','Ubuntu Mono','Courier New',monospace;
  direction: ltr;
  text-align: left;
  white-space: pre;
  word-spacing: normal;
  word-break: normal;
  line-height: 1.5;
  -moz-tab-size: 4;
  -o-tab-size: 4;
  tab-size: 4;
  -webkit-hyphens: none;
  -moz-hyphens: none;
  -ms-hyphens: none;
  -webkit-hyphens: none;
  -moz-hyphens: none;
  -ms-hyphens: none;
  hyphens: none;
}

.c9 pre[class*='language-']::selection,
.c9 code[class*='language-']::selection,
.c9 pre[class*='language-'] *::selection,
.c9 code[class*='language-'] *::selection {
  text-shadow: none;
  background: #75a7ca;
}

.c9 pre[class*='language-'] {
  padding: 1em;
  margin: 0.5em 0;
  overflow: auto;
  background: #1e1e1e;
}

.c9:not(pre) > code[class*='language-'] {
  padding: 0.1em 0.3em;
  border-radius: 0.3em;
  color: #db4c69;
  background: #f9f2f4;
}

.c9 .namespace {
  opacity: 0.7;
}

.c9 .token.doctype .token.doctype-tag {
  color: #569cd6;
}

.c9 .token.doctype .token.name {
  color: #9cdcfe;
}

.c9 .token.comment,
.c9 .token.prolog {
  color: #6a9955;
}

.c9 .token.punctuation,
.c9 .language-html .language-css .token.punctuation,
.c9 .language-html .language-javascript .token.punctuation {
  color: #d4d4d4;
}

.c9 .token.property,
.c9 .token.tag,
.c9 .token.boolean,
.c9 .token.number,
.c9 .token.constant,
.c9 .token.symbol,
.c9 .token.inserted,
.c9 .token.unit {
  color: #b5cea8;
}

.c9 .token.selector,
.c9 .token.attr-name,
.c9 .token.string,
.c9 .token.char,
.c9 .token.builtin,
.c9 .token.deleted {
  color: #ce9178;
}

.c9 .language-css .token.string.url {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

.c9 .token.operator,
.c9 .token.entity {
  color: #d4d4d4;
}

.c9 .token.operator.arrow {
  color: #569cd6;
}

.c9 .token.atrule {
  color: #ce9178;
}

.c9 .token.atrule .token.rule {
  color: #c586c0;
}

.c9 .token.atrule .token.url {
  color: #9cdcfe;
}

.c9 .token.atrule .token.url .token.function {
  color: #dcdcaa;
}

.c9 .token.atrule .token.url .token.punctuation {
  color: #d4d4d4;
}

.c9 .token.keyword {
  color: #569cd6;
}

.c9 .token.keyword.module,
.c9 .token.keyword.control-flow {
  color: #c586c0;
}

.c9 .token.function,
.c9 .token.function .token.maybe-class-name {
  color: #dcdcaa;
}

.c9 .token.regex {
  color: #d16969;
}

.c9 .token.important {
  color: #569cd6;
}

.c9 .token.italic {
  font-style: italic;
}

.c9 .token.constant {
  color: #9cdcfe;
}

.c9 .token.class-name,
.c9 .token.maybe-class-name {
  color: #4ec9b0;
}

.c9 .token.console {
  color: #9cdcfe;
}

.c9 .token.parameter {
  color: #9cdcfe;
}

.c9 .token.interpolation {
  color: #9cdcfe;
}

.c9 .token.punctuation.interpolation-punctuation {
  color: #569cd6;
}

.c9 .token.boolean {
  color: #569cd6;
}

.c9 .token.property,
.c9 .token.variable,
.c9 .token.imports .token.maybe-class-name,
.c9 .token.exports .token.maybe-class-name {
  color: #9cdcfe;
}

.c9 .token.selector {
  color: #d7ba7d;
}

.c9 .token.escape {
  color: #d7ba7d;
}

.c9 .token.tag {
  color: #569cd6;
}

.c9 .token.tag .token.punctuation {
  color: #808080;
}

.c9 .token.cdata {
  color: #808080;
}

.c9 .token.attr-name {
  color: #9cdcfe;
}

.c9 .token.attr-value,
.c9 .token.attr-value .token.punctuation {
  color: #ce9178;
}

.c9 .token.attr-value .token.punctuation.attr-equals {
  color: #d4d4d4;
}

.c9 .token.entity {
  color: #569cd6;
}

.c9 .token.namespace {
  color: #4ec9b0;
}

.c9 pre[class*='language-javascript'],
.c9 code[class*='language-javascript'],
.c9 pre[class*='language-jsx'],
.c9 code[class*='language-jsx'],
.c9 pre[class*='language-typescript'],
.c9 code[class*='language-typescript'],
.c9 pre[class*='language-tsx'],
.c9 code[class*='language-tsx'] {
  color: #9cdcfe;
}

.c9 pre[class*='language-css'],
.c9 code[class*='language-css'] {
  color: #ce9178;
}

.c9 pre[class*='language-html'],
.c9 code[class*='language-html'] {
  color: #d4d4d4;
}

.c9 .language-regex .token.anchor {
  color: #dcdcaa;
}

.c9 .language-html .token.punctuation {
  color: #808080;
}

.c9 pre[data-line] {
  position: relative;
}

.c9 pre[class*='language-'] > code[class*='language-'] {
  position: relative;
  z-index: 1;
}

.c9 .line-highlight {
  position: absolute;
  left: 0;
  right: 0;
  padding: inherit 0;
  margin-top: 1em;
  background: #f7ebc6;
  box-shadow: inset 5px 0 0 #f7d87c;
  z-index: 0;
  pointer-events: none;
  line-height: inherit;
  white-space: pre;
}

.c10 {
  width: 100%;
  font-size: calc(1.6rem + 0.2rem);
  line-height: 1.5;
}

.c10 iframe[src*='ads'] {
  display: none;
}

.c10 p {
  width: 100%;
  margin: 2.4rem 0;
  word-wrap: break-word;
}

.c10 a,
.c10 a:visited,
.c10 a:link {
  color: #dc143c;
  -webkit-text-decoration: none;
  text-decoration: none;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c10 a:hover {
  -webkit-filter: brightness(50%);
  filter: brightness(50%);
}

.c10 code {
  font-family: monospace;
  color: #dc143c;
  font-size: 1.6rem;
  background: #DDDDDD;
  padding: 0.2rem;
  margin: 0.2rem;
}

.c10 pre {
  background: #000000;
  padding: 2.4rem;
  font-family: monospace;
  color: #FFFFFF;
  margin: 2.4rem 0;
  overflow-x: auto;
  font-size: 1.6rem;
}

.c10 pre code {
  color: inherit;
  background: inherit;
}

.c10 img {
  max-width: 100%;
}

.c10 .image {
  background: #DDDDDD;
  line-height: 0;
  margin: 2.4rem 0;
}

.c10 .image figcaption {
  font-size: 1.6rem;
  padding: 1.6rem 0;
  text-align: center;
  line-height: 1.3;
  margin: 2.4rem 0;
}

.c10 .image-style-side {
  max-width: 50%;
  float: right;
  margin: 2.4rem;
}

.c10 hr {
  border: none;
  border: 1px solid red;
  border-bottom: 0.1rem solid #DDDDDD;
}

.c10 ul,
.c10 ol {
  margin: 2.4rem 4.0rem;
}

.c10 .table {
  width: 100%;
  overflow: hidden;
  overflow-x: auto;
}

.c10 table {
  width: 100%;
  border-collapse: collapse;
}

.c10 table td,
.c10 table th {
  padding: 1.6rem;
  border: 0.1rem solid #DDDDDD;
}

.c10 blockquote {
  border-left: 0.5rem solid #dc143c;
  color: #AAAAAA;
  -webkit-filter: brightness(80%);
  filter: brightness(80%);
  padding-left: 2.4rem;
  font-style: italic;
  margin: 2.4rem;
}

.c1 {
  width: 100%;
  margin: 0 auto;
  padding: 0 1.6rem;
  max-width: 96rem;
}

.c8 {
  width: 100%;
  margin: 0 auto;
  padding: 0 1.6rem;
  max-width: 80rem;
}

.c0 {
  width: 100%;
  padding-left: 1.6rem;
}

@media (max-width:768px) {
  .c4 {
    font-size: 4.0rem;
  }
}

@media print {
  .c9 pre[class*='language-'],
  .c9 code[class*='language-'] {
    text-shadow: none;
  }
}

@media (max-width:768px) {
  .c10 {
    font-size: 2rem;
  }

  .c10 pre {
    width: 82%;
  }

  .c10 .image-style-side {
    max-width: 100%;
    float: none;
    margin: 0;
  }
}

<div>
  <section
    class="c0"
  >
    <section
      class="c1"
    >
      <section
        class="c2"
      >
        <h2
          class="c3 c4"
        >
          Lorem Ipsum is simply dummy text of the printing and typesetting industry.
        </h2>
        <p
          class="c5"
        >
          Lorem Ipsum has been the industry's standard dummy text ever since the 1500s,
        </p>
        <img
          alt="Lorem Ipsum is simply dummy text of the printing and typesetting industry."
          class="c6"
          src="https://res.cloudinary.com/project-cdn/image/upload/v1643992684/use_Effect_hook_62405844a5.png"
        />
        <div
          class="c7"
        >
          <p>
            <span>
              Por 
            </span>
            <a
              href="/authors/luan-mateus"
            >
              Luan Mateus
            </a>
            <span
              class="separator"
            >
               | 
            </span>
            <time
              datetime="2021-03-02T15:16:09.246+00:00"
            >
              2 de mar. de 2021
            </time>
            <span
              class="separator"
            >
               | 
            </span>
            <span
              class="categories"
            >
              <span>
                <a
                  href="/categories/tech"
                >
                  Tech
                </a>
              </span>
              <span>
                <a
                  href="/categories/javascript"
                >
                  JS
                </a>
              </span>
            </span>
          </p>
        </div>
      </section>
    </section>
    <section
      class="c8"
    >
      <div
        class="c9"
      >
        <div
          class="c10"
        >
          
  
          <p>
            Nostro 
            <strong>
              definitiones
            </strong>
             (
            <i>
              React
            </i>
            ) sed cu - 
            <u>
              Ot√°vio Miranda
            </u>
            . Sea 
            <code>
              useEffect
            </code>
            , useState dicat fabulas 
            <a
              href="https://gist.github.com/luizomf/e7cf0c3d172801225cce694681ecb0ad"
            >
              consetetur
            </a>
             no, odio deseruisse assueverit mei eu, vis probo nonumes molestie id. Ipsum movet liberavisse an mea, diceret vituperata ne his, delenit apeirian quo ea. Imperdiet percipitur te eos. Per ludus aliquam cu. 
            <a
              href="https://gist.github.com/luizomf/e7cf0c3d172801225cce694681ecb0ad"
            >
              Eos ex tincidunt necessitatibus, timeam theophrastus mel cu
            </a>
            . La.
          </p>
          
  
          <h1>
            Tincidunt necessitatibus, timeam theophrastus mel cu
          </h1>
          
  
          <h2>
            Tincidunt necessitatibus, timeam theophrastus mel cu
          </h2>
          
  
          <h3>
            Tincidunt necessitatibus, timeam theophrastus mel cu
          </h3>
          
  
          <h4>
            Tincidunt necessitatibus, timeam theophrastus mel cu
          </h4>
          
  
          <h5>
            Tincidunt necessitatibus, timeam theophrastus mel cu
          </h5>
          
  
          <h6>
            Tincidunt necessitatibus, timeam theophrastus mel cu
          </h6>
          
  
          <pre
            class="language-plaintext"
            tabindex="0"
          >
            <code
              class="language-plaintext"
            >
              import { HtmlContent } from '../HtmlContent';
  import * as Styled from './styles';
  export type FooterProps = {
    footerHtml: string;
  };
  export const Footer = ({ footerHtml }: FooterProps) =&gt; {
    return (
      &lt;Styled.Container&gt;
        &lt;HtmlContent html={footerHtml} /&gt;
      &lt;/Styled.Container&gt;
    );
  };
  
            </code>
          </pre>
          
  
          <p>
            Qui eu 
            <i>
              mutat
            </i>
             exerci 
            <span
              style="background-color:hsl(60,75%,60%);"
            >
              dolores
            </span>
            , his no 
            <span
              style="color:hsl(270,75%,60%);"
            >
              repudiare
            </span>
             constituam 
            <s>
              accommodare
            </s>
            , vis ea sint 
            <u>
              tempor
            </u>
             eirmod. In mei eruditi moderatius, ex per iudico essent repudiandae. In vis vidisse menandri, an has repudiare disputationi delicatissimi. In has wisi justo postulant. Ei viderer ornatus mel, usu an partiendo tincidunt, ea eum brute ullamcorper.
          </p>
          <p>
            10
            <sup>
              2
            </sup>
            *5
            <sup>
              2
            </sup>
             - H
            <sub>
              2
            </sub>
            0
          </p>
          <p
            style="text-align:center;"
          >
            <strong>
              Center: 
            </strong>
            Abhorreant reformidans eum an, per equidem repudiandae no, ex cum elitr argumentum persequeris. In eam consul theophrastus delicatissimi, lobortis rationibus efficiendi mea ei, nam natum dolore scripserit ea. Sit partiendo eloquentiam ne. Sea no autem dolores, ridens gloriatur mel cu. Munere nullam epicuri his id, dicam definiebas deterruisset eu vim. Ea veritus deleniti usu, nam ei idque choro volumus, ex nisl aliquam bonorum eam.
          </p>
          <p>
            Fugit aliquip erroribus eu eam, an mel vidit doctus. Vis nobis veritus maluisset id, nonumy adversarium neglegentur est ut. Tation persius pri eu, modo minim fastidii mei no. Ad tota pertinax definiebas eam.
          </p>
          <p
            style="text-align:right;"
          >
            <strong>
              Right:
            </strong>
             Mel in lorem feugiat, tation dolore et eum. Id has utamur albucius molestiae. Dico justo maiestatis ei mei, labitur laboramus voluptatum nam ut. Saperet propriae nec an.
          </p>
          <p>
            Quod oblique mei ne, cetero delicatissimi sea cu, cu est harum dicit verterem. Tota forensibus ut usu, luptatum dignissim persecuti vel ex. Mea nostrud instructior cu, et referrentur theophrastus eum. Pro in iuvaret senserit. In persius explicari conceptam nec, in populo ponderum consequat vis. Gloriatur conclusionemque no qui, vitae nominati nec te, laoreet torquatos accommodare ex qui. Eu natum mediocritatem eos, cum ex alia oportere.
          </p>
          <p
            style="text-align:justify;"
          >
            <strong>
              Justify:
            </strong>
             Cu vis nonumes petentium disputationi, invidunt aliquando ex ius, ocurreret suscipiantur conclusionemque eu qui. Probo nemore ut est. Et eam dicta oratio eligendi. Quem dicta dolore vel et, at reque gloriatur definitionem sed.
          </p>
          <p>
            Nec simul ullamcorper at. Mea possit dolores persecuti te, ei quidam apeirian quo. Has quas corpora ne. Ius ad malis iusto delenit, sit ferri idque ex.
          </p>
          <blockquote>
            <p>
              Blockquote: Et sed legere rationibus, vitae evertitur ad eam. Eu amet vero officiis vis. Ad nullam pericula neglegentur sit, vix ei regione legendos argumentum. Mei an quas convenire, nobis dicunt mei ut. Pro no debet referrentur, ei est falli libris. Eu per summo salutandi hendrerit, salutatus democritum interesset nec ut. Wisi lobortis incorrupte ei mea, explicari interpretaris has et.
            </p>
          </blockquote>
          <p>
            A new line:
          </p>
          <hr />
          <p>
            Now keep going‚Ä¶
          </p>
          <figure
            class="image"
          >
            <a
              href="https://gist.github.com/luizomf/e7cf0c3d172801225cce694681ecb0ad"
            >
              <img
                alt="O alt text da imagem"
                src="https://res.cloudinary.com/dlizakp2a/image/upload/v1614696630/Frozen_sunset_on_the_lake_by_Manuel_Arslanyan_9f9cd8ea10.jpg"
              />
            </a>
            <figcaption>
              Aqui est√° uma imagem
            </figcaption>
          </figure>
          <p>
            Munere euripidis an sed, usu ne consul soluta utamur. Bonorum equidem placerat eu cum, tantas quidam duo et. Pro soleat quaestio ne, ut nam wisi vocibus inciderint, ex his maiorum mnesarchum. Et est wisi primis, id vel odio alienum voluptatibus. Veri doctus offendit vim ea. Has nemore prompta inciderint ei, no sea expetendis dissentiet. Copiosae delicatissimi an sea, id cum agam clita nostro, tale tincidunt delicatissimi ei eos.
          </p>
          <figure
            class="image image-style-side"
          >
            <img
              alt="Uma imagem √† direita"
              src="https://res.cloudinary.com/dlizakp2a/image/upload/v1614696630/Frozen_sunset_on_the_lake_by_Manuel_Arslanyan_9f9cd8ea10.jpg"
            />
            <figcaption>
              Uma imagem √† direita
            </figcaption>
          </figure>
          <p>
            Munere euripidis an sed, usu ne consul soluta utamur. Bonorum equidem placerat eu cum, tantas quidam duo et. Pro soleat quaestio ne, ut nam wisi vocibus inciderint, ex his maiorum mnesarchum. Et est wisi primis, id vel odio alienum voluptatibus. Veri doctus offendit vim ea. Has nemore prompta inciderint ei, no sea expetendis dissentiet. Copiosae delicatissimi an sea, id cum agam clita nostro, tale tincidunt delicatissimi ei eos.
          </p>
          <p>
            Munere euripidis an sed, usu ne consul soluta utamur. Bonorum equidem placerat eu cum, tantas quidam duo et. Pro soleat quaestio ne, ut nam wisi vocibus inciderint, ex his maiorum mnesarchum. Et est wisi primis, id vel odio alienum voluptatibus. Veri doctus offendit vim ea. Has nemore prompta inciderint ei, no sea expetendis dissentiet. Copiosae delicatissimi an sea, id cum agam clita nostro, tale tincidunt delicatissimi ei eos.
          </p>
          <p>
            Solum consul scripserit in duo, an sit postea denique percipit. Facer ignota consequat nec ei, idque etiam corrumpit sit ut. In mucius definitionem quo, vis an nisl diam appellantur, hendrerit comprehensam ea per. Vix saepe fierent ut.
          </p>
          <p>
            Cu mei summo copiosae. Et vim quaestio partiendo. Quot praesent imperdiet at duo. Sit probo causae ei. Ad ius nemore eruditi.
          </p>
          <p>
            Mei in qualisque disputando. Adipisci philosophia et sed, mel diceret patrioque sententiae ea, iriure posidonium consectetuer ad eam. Nobis reprimique duo in, eam corpora mandamus ex. In quod euismod eum, ne habemus cotidieque necessitatibus cum. Liber recteque duo no, ludus reprehendunt has an.
          </p>
          <p>
            Solum consul scripserit in duo, an sit postea denique percipit. Facer ignota consequat nec ei, idque etiam corrumpit sit ut. In mucius definitionem quo, vis an nisl diam appellantur, hendrerit comprehensam ea per. Vix saepe fierent ut.
          </p>
          <p>
            Cu mei summo copiosae. Et vim quaestio partiendo. Quot praesent imperdiet at duo. Sit probo causae ei. Ad ius nemore eruditi.
          </p>
          <p>
            Mei in qualisque disputando. Adipisci philosophia et sed, mel diceret patrioque sententiae ea, iriure posidonium consectetuer ad eam. Nobis reprimique duo in, eam corpora mandamus ex. In quod euismod eum, ne habemus cotidieque necessitatibus cum. Liber recteque duo no, ludus reprehendunt has an.
          </p>
          <p>
            Bullet:
          </p>
          <ul>
            <li>
              Mei in qualisque disputando. Adipisci philosophia et sed, mel diceret patrioque sententiae ea, iriure posidonium consectetuer ad eam. Nobis reprimique duo in, eam corpora mandamus ex. In quod euismod eum, ne habemus cotidieque necessitatibus cum. Liber recteque duo no, ludus reprehendunt has an.
            </li>
            <li>
              iriure posidonium consectetuer ad eam.
            </li>
            <li>
              Nobis reprimique duo in, eam corpora mandamus
            </li>
            <li>
              ex. In
            </li>
            <li>
              quod euismod
            </li>
            <li>
              eum, ne habemus cotidieque necessitatibus cum.
            </li>
            <li>
              Liber recteque duo no, ludus reprehendunt has an.
            </li>
          </ul>
          <p>
            Now ordered:
          </p>
          <ol>
            <li>
              Item 1
            </li>
            <li>
              Item
            </li>
            <li>
              Item
            </li>
            <li>
              Item
            </li>
          </ol>
          <p>
            A think that's it‚Ä¶
          </p>
          <figure
            class="table"
          >
            <table>
              <thead>
                <tr>
                  <th>
                    Title 1
                  </th>
                  <th>
                    Title 1
                  </th>
                  <th>
                    Title 1
                  </th>
                  <th>
                    Title 1
                  </th>
                  <th>
                    Title 1
                  </th>
                  <th>
                    Title 1
                  </th>
                  <th>
                    Title 1
                  </th>
                  <th>
                    Title 1
                  </th>
                  <th>
                    Title 1
                  </th>
                  <th>
                    Title 1
                  </th>
                </tr>
              </thead>
              <tbody>
                <tr>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                </tr>
                <tr>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                </tr>
                <tr>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                  <td>
                    Item
                  </td>
                </tr>
              </tbody>
            </table>
          </figure>
          <p>
            A video:
          </p>
          <figure
            class="media"
          >
            <oembed
              url="https://www.youtube.com/watch?v=oLmRV25eQvo"
            />
          </figure>
          <p>
            The end.
          </p>
          
  
        </div>
      </div>
    </section>
  </section>
</div>
`;
